lib/Twig/Extensions/Extension/Date.php

```diff
@@ -101,7 +101,7 @@ protected function getPluralizedInterval($count, $invert, $unit)
             return $this->translator->transChoice($id, $count, array('%count%' => $count), 'date');
         }
 
-        if ($count > 1) {
+        if (1 !== $count) {
```

Comment:
@nguyentienlong the extension will never pass a negative value. The only way to get a negative value here is to extend the class and mess with the protected method directly (as it is not private, we cannot be 100% sure indeed). I would consider it a bug in your own code then. The usage of ``$this->translator->transChoice`` also assumes it is not negative (translation pluralization rules deal with quantities, which are non negative integers)